### ISSUE TITLE
Enhance generator to indicate how a YANG module was processed 

### ISSUE THREAD

**Issue by 111pontes** [CiscoDevNet/ydk-gen#894]
Some of the input YANG modules may not contain data nodes (e.g. type definitions).  Some other YANG modules may define data nodes, but be used as augmentation.  As the number of input YANG files typically does not match the number of output files, the generator should report how each input file was processed and provide the total number of input files and the total number of output files.   I'd suggest to produce the total count as normal output and the detailed description per file as verbose (logging) output.

**Comment by ygorelik** [comment]
Implemented in 0.8.4.post1 by adding debug statements, which are enabled with '-v' option.
Examples of log:
```
Created models archive: /Users/ygorelik/ydk-gen/gen-api/python/ydktest-bundle/ydktest@0.1.0.tar.gz
Processing 1 of 45 /Users/ygorelik/ydk-gen/gen-api/.cache/models/ydktest@0.1.0/iana-crypt-hash@2014-08-06.yang
    Skipping module, because it does not contain top level containers
Processing 2 of 45 /Users/ygorelik/ydk-gen/gen-api/.cache/models/ydktest@0.1.0/iana-if-type.yang
    Printing python module /Users/ygorelik/ydk-gen/gen-api/python/ydktest-bundle/ydk/models/ydktest/iana_if_type.py
Processing 3 of 45 /Users/ygorelik/ydk-gen/gen-api/.cache/models/ydktest@0.1.0/ietf-aug-base-1@2016-07-01.yang
    Printing python module /Users/ygorelik/ydk-gen/gen-api/python/ydktest-bundle/ydk/models/ydktest/ietf_aug_base_1.py
```

**Comment by ygorelik** [comment]
Resolved the issue in private fork: [af672f2c](https://github.com/ygorelik/ydk-gen/commit/af672f2cb0938ad3930600f92caf16312c600923)
